tools/test/test262/template.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <excludeList>
     <!-- exclude each test file -->
+    <test id="built-ins/decodeURI/S15.1.3.1_A2.5_T1"><reason>TOO SLOW</reason></test>
+    <test id="built-ins/decodeURI/S15.1.3.1_A2.5_T1"><reason>TOO SLOW</reason></test>
+    <test id="built-ins/decodeURIComponent/S15.1.3.2_A2.5_T1"><reason>TOO SLOW</reason></test>
+    <test id="built-ins/decodeURIComponent/S15.1.3.2_A2.5_T1"><reason>TOO SLOW</reason></test>
```

Comment:
Our test262 driver supports fast/slow mode.
In slow mode, it runs **slow** TCs marked as `SLOW` not `TOO SLOW`.
Currently, Jenkins CI executes test262 with slow mode in `release-64bit-test262` stage.
(You can also find mode option at [here](https://github.com/Samsung/escargot/blob/9f8b6fdb04deecc30b23a2a35c284f0aa64779ba/tools/run-tests.py#L184))

Reply:
I am going to rename from TOO SLOW to SLOW